### PR TITLE
 does not display title only if title not equals to video

### PR DESCRIPTION
**Pull request by nikhedonia**
* does display title if title does not equals to video
* refactors videolabel so it does not need platform dependent
implementations

* fixes misplaced pipe on andorid (move up by one pixel)

### CODE REVIEW

File: packages/video-label/beautify-title.android.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// apply transformations to add uppercase and letter spacing.
+// letterSpacing CSS prop does not work on android:
+// https://github.com/facebook/react-native/pull/13199
+
+export default title =>
```
Comment (colincclark) [inline packages/video-label/beautify-title.android.js]: we should unit test all helper functions

File: packages/video-label/video-label.js [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
   <View style={{ flexDirection: "row", alignItems: "center" }}>
     <IconVideo height={styles.title.fontSize} fillColor={color} />
-    <Text style={[styles.title, { color, marginLeft: 5 }]}>VIDEO</Text>
-    {title ? <Text style={[styles.pipe, { marginLeft: 5 }]}>|</Text> : null}
-    {title ? (
-      <Text style={[styles.title, { color, paddingLeft: 5 }]}>
-        {title.toUpperCase()}
-      </Text>
+    <Text style={[styles.title, { color, marginLeft: 5 }]}>
+      {beautifyTitle("VIDEO")}
```
Comment (colincclark) [inline packages/video-label/video-label.js]: does this uppercase word need to be uppercased?

**Comment by colincclark** [comment]
this still needs tests?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/c01f88c50c2877a47bac09ac594684bdfb3bd658/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/7fb63c30a1a8eadc23a28c1c05fdb4573489b378/index.html

**Pull request merged**